components/Picto/Picto.ios.tsx

```diff
@@ -16,12 +16,12 @@ export default function Picto({ picto, style, size, weight, color }: Props) {
     <SFSymbol
       name={picto}
       weight={(weight as any) || "regular"}
-      scale="large"
+      // scale="large"
```

Comment:
I'm not sure why we had this. Removed it here and in my navigation refactor. We'll need to make sure all icons are okay but I think it's better to have a 1-1 with Android for sizing